src/controllers/simplified-account/settings/api-keys/api-keys.controller.test.js

```diff
@@ -47,6 +48,13 @@ describe('Controller: settings/api-keys', () => {
             revokeKeyLink: `/simplified/service/${SERVICE_ID}/account/${ACCOUNT_TYPE}/settings/api-keys/revoke/${apiKeys[0].tokenLink}`
           }
         }))
+      expect(mockResponse.args[0][3]).to.have.property('createApiKeyLink').to.equal(
+        `/simplified/service/${SERVICE_ID}/account/${ACCOUNT_TYPE}/settings/api-keys/create`
+      )
+      expect(mockResponse.args[0][3]).to.have.property('revokedKeysLink').to.equal(
+        `/simplified/service/${SERVICE_ID}/account/${ACCOUNT_TYPE}/settings/api-keys/revoked`
+      )
+      expect(mockResponse.args[0][3]).to.have.property('showRevokedKeysLink').to.equal(false)
```

Comment:
Another scenario would be for `showRevokedKeysLink` to be `true` if `apiKeysService.getRevokedKeys` returns a list, but this will be implicitly covered in the cypress test.